docs/marine-book/marine-tooling-reference/marine-repl.md

```diff
@@ -151,12 +151,15 @@ elapsed time: 421.775µs
 
 ### call: call a function
 
-Allows you to call the specified module. Please note that from Aqua only the `facade` module could be called. To provide several arguments to a function make them separated by a space in square brackets
+Allows you to call the specified module's function. Please note that from Aqua only the `facade` module could be called. To provide several arguments to the function, pass them as a JSON array.
 
 ```
 1> call greeting greeting "Fluence"
 result: String("Hi, Fluence")
  elapsed time: 160.923µs
+2> call avm invoke ["(null)", [], [], {"init_peer_id": "init", "current_peer_id": "some", "timestamp": 1500000000000, "ttl": 120000}, [123, 125]]
+result: ...
+ elapsed time: 45.712375ms
```

Comment:
Agree with changes above, but this one is confusing. How should user know what is "avm"? Maybe it is better to put it into aquavm's docs. Plus, its quite rare to see a greeting module and aquavm module both loaded into one repl.

Reply:
Perhaps, the AquaVM example should be in a separate paragraph.  Anyway, it is the example of multi-argument call, that can be useful for those who runs AquaVM manually.

Reply:
I think that it's worth having a link to the interpreter interaction [scheme](https://github.com/fluencelabs/aquavm/tree/master/air), otherwise, it's tough to guess what are these parameters.